mteb/tasks/Retrieval/multilingual/CUREv1Retrieval.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+from enum import Enum
+
+from datasets import DatasetDict, load_dataset
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+from ....abstasks.MultilingualTask import MultilingualTask
+
+_LANGUAGES = {
+    "en": ["eng-Latn", "eng-Latn"],
+    "es": ["spa-Latn", "eng-Latn"],
+    "fr": ["fra-Latn", "eng-Latn"],
+}
+
+
+class CUREv1Splits(str, Enum):
+    all = "All"
+    dentistry_and_oral_health = "Dentistry and Oral Health"
+    dermatology = "Dermatology"
+    gastroenterology = "Gastroenterology"
+    genetics = "Genetics"
+    neuroscience_and_neurology = "Neuroscience and Neurology"
+    orthopedic_surgery = "Orthopedic Surgery"
+    otorhinolaryngology = "Otorhinolaryngology"
+    plastic_surgery = "Plastic Surgery"
+    psychiatry_and_psychology = "Psychiatry and Psychology"
+    pulmonology = "Pulmonology"
+
+    @classmethod
+    def names(cls) -> list[str]:
+        return sorted(cls._member_names_)
+
+
+class CUREv1Retrieval(MultilingualTask, AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        dataset={
+            "path": "clinia/CUREv1",
+            "revision": "3bcf51c91e04d04a8a3329dfbe988b964c5cbe83",
+        },
+        name="CUREv1",
+        description="Collection of query-passage pairs curated by medical professionals, across 10 disciplines and 3 cross-lingual settings.",
+        type="Retrieval",
+        modalities=["text"],
+        category="s2p",
+        reference="https://huggingface.co/datasets/clinia/CUREv1",
+        eval_splits=CUREv1Splits.names(),
+        eval_langs=_LANGUAGES,
+        main_score="ndcg_at_10",
+        date=("2024-01-01", "2024-10-31"),
+        domains=["Medical", "Academic"],
```

Comment:
```suggestion
        domains=["Medical", "Academic", "Written"],
```
Assumed it is not spoken